packages/react-pagination/lib/react-pagination.js

```diff
@@ -0,0 +1,147 @@
+import { Box, BoxBody } from "@wpmudev/react-box";
```

Comment:
@Gowtham369 I noticed you added this component and even when I recommend in another review to remove it because we won't need it, I noticed also that is not currently added to the `package.json` file. Please, take a closer look at the documentation since this happened because you didn't include the package in the correct way.

**Note:** This is not a fix request, just some heads-up for future releases.